modules

```diff
@@ -9,5 +9,5 @@
 GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
-PACKAGES_COMMUNITY_COMMIT=ed68c7060e6122537038ece7de920d199aeed9b0
+PACKAGES_COMMUNITY_COMMIT=61ba5ab54ffd34a77ec4967487f69848d553fcc0
```

Comment:
Shouldn't this be backported from the original PR? I assume we missed to backport some PR here.

Reply:
the update was in this pr: https://github.com/freifunkMUC/site-ffm/pull/478
but we dont use `ffac-mt7915e-hotfix` in stable

Reply:
fair enough. As long as they are identical now, we can continue backporting next time.